### DISCUSSION TITLE
can i put a pride flag

### DISCUSSION THREAD

**Discussion by raynepaws** [cytrinc/bark-coding#203] [Q&A] [answered]
just a lil somethin for pride month 🏳️‍🌈 🏳️‍⚧️ !!
![Screenshot from 2024-06-01 00-43-39](https://github.com/mariocraft987/bark-coding/assets/129226914/7f8dd0eb-17d2-411e-a34e-7406d69365b2)

**Comment by mariocraft987** [comment, marked as answer]
k
